.circleci/config.yml

```diff
@@ -43,6 +43,7 @@ jobs:
 
       - run:
           name: Install go dependencies
-          command: go get -u -v github.com/golang/protobuf/protoc-gen-go
+          command: go get -u -v github.com/golang/protobuf/protoc-gen-go;
```

Comment:
list the multiple packages as multiple arguments to the same 'go get' command.

Reply:
done